deeplake/api/dataset.py

```diff
@@ -81,6 +81,18 @@
 from deeplake.core.storage.deeplake_memory_object import DeepLakeMemoryObject
 
 
+def _check_indra_and_read_only_flags(indra: bool, read_only: Optional[bool]):
+    if indra == False:
+        return
+    if read_only == True:
+        return
+    if read_only is None:
+        raise ValueError(
+            "'indra = True' is only available for read_only datasets. Please also specify 'read_only = True'."
+        )
+    raise ValueError("'indra = True' is only available for read_only datasets.")
```

Comment:
@khustup2 can we also add the solution to this line `Please also specify 'read_only = True'.`